New package: dJUICE v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: dJUICE
- Repository: https://github.com/DJ4Earth/dJUICE.jl
- Created by: @enigne
- Version: v0.1.0
- Commit: 0fe8dfb944492a3528c8ad10b4885801ea553ca4
- Git reference: HEAD
- Description: differentiable JUlia ICE model
- Release notes:
<!-- BEGIN RELEASE NOTES -->
> add to register
<!-- END RELEASE NOTES -->

goerz commented:
[noblock] I’m not sure the lowercase first letter is OK for a Julia package. In addition, could you add a paragraph to the README to give some context for the package?

JuliaHubRegistrator committed:
New package: dJUICE v0.1.0

UUID: 1cb5fe60-6b70-416b-9050-039c23a62a10
Repo: https://github.com/DJ4Earth/dJUICE.jl.git
Tree: 59522bc5d22078848ff9786baf0945fa32bf16d5

Registrator tree SHA: 7242ef64be5953fac9bf8b3efa3a7c4d2d44ca09

enigne commented:
Hi, we have added descriptions of the package in README and changed the name to start with uppercase. Please help us to merge the package. Thank you.

goerz commented:
You will have to open a new registration for the new name

goerz commented:
Follow-up at https://github.com/JuliaRegistries/General/pull/107805
